### DISCUSSION TITLE
Consider synonyms "ATTR-CM" "hereditary transthyretin-mediated amyloidosis" for MONDO:0019441

### DISCUSSION THREAD

**Discussion by kshefchek** [monarch-initiative/mondo#3870] [Bigger project]
**Mondo term (ID and Label)**
MONDO:0019441, ATTRV122I amyloidosis 

**Synonym to be added**
ATTR-CM
hereditary transthyretin-mediated amyloidosis (could also be primary label)

**Synonym subtype: ie, broad/exact/narrow/related**
Broad - ATTR-CM
Broad - transthyretin-mediated amyloidosi
Exact - hereditary transthyretin-mediated amyloidosis

**Database cross reference for the synonym** Such as PubMed ID (in the format PMID:XXXXXX) or a cross-reference to another ontology, like OMIM or Orphanet.

https://www.heart.org/-/media/files/health-topics/answers-by-heart/abh_what-is-attrcm_v2_a

**Comment by kshefchek** [comment]
this might instead apply to MONDO:0017132 'hereditary ATTR amyloidosis'

**Comment by nicolevasilevsky** [comment]
@pnrobinson do you know which Mondo term is synonymous for Transthyretin Amyloid Cardiomyopathy?

I think these synonyms should be applied to MONDO:0017132 'hereditary ATTR amyloidosis' but I'm not totally sure. Thanks!

**Comment by pnrobinson** [comment]
ATTR amyloid is essentially the same thing as Transthyretin Amyloid.
However, not all ATTR amyloidosis is hereditary/Mendelian. Also, the amyloid is not necessarily limited to the heart nor does it always cause heart dysfunction (cardiomyopathy). So there are two reasons why these items are not synonymous.

The next question is how to represent amyloid diseases.

I am not an expert, but I think the basics are

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6989279/  (hereditary and acquired Transthyretin Amyloid Cardiomyopathy)


Kidney amyloidosis
https://cjasn.asnjournals.org/content/7/8/1337

Other organs can be affected. 

We could represent the disease as "Amyloidosis" and have the specific organ involvement of a patient be HPO terms, or we could represent the involvement of each organ as a separate disease (one patient might then have two different amyloid diseases, e.g., for heart and kidney).

**Comment by nicolevasilevsky** [comment]
Need to discuss with @mellybelly and @cmungall and @ahamosh - we should improve the classification of amyloidosis (ie organ affected, proteins involved, etc.)

**Comment by nicolevasilevsky** [comment]
need better documentation on how we are classifying these hierarchies. The patterns are meant to (partially) address this.

**Comment by nicolevasilevsky** [comment]
@kshefchek do you mind if we hold off on adding these synonyms? We need to restructure this hierarchy.

**Comment by ahamosh** [comment]
I am not (at all) an expert on amyloidosis, but I think the trouble is that it can address multiple organs (or just one) and can be hereditary or acquired and there are different etiologies.  TTR is important because it now has a specific treatment.  I did a very quick search in OMIM and there are 41 amyloidoses with clinical synopses.  It would take me a while to sort through all this...
